.golangci.yml

```diff
@@ -9,6 +9,7 @@ linters:
     - deadcode
     - depguard
     - dogsled
+    # - errcheck
```

Comment:
commented out for now since there are more issues that should be solved in another pr